.github/workflows/test.yml

```diff
@@ -27,3 +27,10 @@ jobs:
 
       - name: Test
         run: go test -v ./...
+
+  docker-build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: 'Build Docker image'
+        run: docker build .
```

Comment:
Can you label the built image (like `local-scip-go`) and add another command to test that the image is runnable? Outputting help would be enough IMO

Reply:
Added `docker build` and `docker run` commands.